content/docs/for-app-developers/concepts/experimental-features.md

```diff
@@ -0,0 +1,18 @@
++++
+title="What are experimental features?"
+weight=8
++++
+
+Certain features are considered `experimental` and susceptible to change in a future API version.
+
+<!--more-->
+
+This means users will need to enable the `experimental` mode in order to use one of these feature.
```

Comment:
@AidanDelaney and @natalieparellano can fill in the features that are currently experimental

Reply:
@AidanDelaney still waiting on this

Reply:
The following features are experimental

* Image extensions
* Windows extensions
* Windows containers
* Windows buildpackage
* Buildpack registry
* Flattening a buildpack package

Reply:
Added